client/plots/matrix.interactivity.js

```diff
@@ -413,6 +413,8 @@ function setTermActions(self) {
 				if (tw && !tw.q) throw 'data.q{} missing from pill callback'
 				const t = self.activeLabel || self.lastactiveLabel
 				if (tw) {
+					// users could midify the term label, need to update tw.label to the latest label
+					tw.label = t.label
```

Comment:
because label is not a property of termwrapper, please refactor to keep a mapping from tw.$id to customized label, and do this when accessing it:

printLabel = this.tw$id2Customlabel.get(tw.$id) || tw.term.name

Reply:
that may not get it saved in state. can merge now but should avoid arbitrary change of tw{}